ip_map_branch.txt

```diff
@@ -10,7 +10,7 @@ two_c	https://github.com/openemr/openemr.git	rel-701	0	0	0	0	demo_5_0_0_5.sql	0
 three	https://github.com/openemr/openemr.git	rel-700	0	0	0	0	0	0	1	three.openemr.io	hey	branch	0	2	0	0	Public OpenEMR 7.0.0 Development Demo on Ubuntu 22.04 (with nodejs 16)
 three_a	https://github.com/openemr/openemr.git	rel-700	0	0	0	0	demo_5_0_0_5.sql	0	1	three.openemr.io/a	hey	branch	5.0.0	2	0	0	Public OpenEMR 7.0.0 Development Demo With Demo Data on Ubuntu 22.04 (with nodejs 16)
 four	https://github.com/omegasystemsgroup/openemr.git	patch-6	0	0	0	0	0	0	1	four.openemr.io	hey	branch	0	0	0	0	UP FOR GRABS Public OpenEMR Demo - Alpha
-four_a	https://github.com/sjpadgett/openemr.git	oefaxsms	0	0	0	0	0	0	1	four.openemr.io/a	hey	branch	0	0	0	0	UP FOR GRABS Public OpenEMR Demo - Beta
+four_a	https://github.com/robertdown/openemr.git	issue-6658	0	0	0	0	demo_5_0_0_5.sql	0	1	four.openemr.io/a	hey	branch	0	0	0	0	UP FOR GRABS Public OpenEMR Demo With Demo Data - Beta
```

Comment:
note if do demo data then also need to put the base version (which is `5.0.0` in the column after `branch`; ie. change the `0` to `5.0.0`)

Reply:
thanks, was just about to abandon demo data :)